Sources/ProjectSpec/Scheme.swift

```diff
@@ -368,6 +412,8 @@ extension Scheme.Run: JSONObjectConvertible {
         region = jsonDictionary.json(atKeyPath: "region")
         debugEnabled = jsonDictionary.json(atKeyPath: "debugEnabled") ?? Scheme.Run.debugEnabledDefault
         simulateLocation = jsonDictionary.json(atKeyPath: "simulateLocation")
+        let maybeStoreKitLocation: String? = jsonDictionary.json(atKeyPath: "storeKitConfiguration")
+        storeKitConfiguration = jsonDictionary.json(atKeyPath: "storeKitConfiguration") ?? maybeStoreKitLocation.flatMap { Scheme.StoreKitConfiguration(location: $0) }
```

Comment:
```suggestion
        storeKitConfiguration = maybeStoreKitLocation ?? maybeStoreKitLocation.flatMap { Scheme.StoreKitConfiguration(location: $0) }
```

Reply:
Though it looks redundant, `jsonDictionary.json(atKeyPath: "storeKitConfiguration")` on L416 is actually attempting to decode "storeKitConfiguration" as a `Scheme.StoreKitConfiguration`, whereas on L415 it's attempting to decode it as a `String`. I've pushed a commit cleaning this up and moving it into `Scheme.StoreKitConfiguration`'s init, let me know what you think!

Reply:
Whoops, got it. Nice, this adjustment looks solid 👍🏻